src/main/resources/application-local.yaml

```diff
@@ -11,3 +11,8 @@ management:
   endpoint:
     health:
       show-details: always
+
+#logging:
+#  level:
+#    org.springframework.data.mongodb: DEBUG
+#    org.mongodb.driver: DEBUG
```

Comment:
should commented code be here?

Reply:
It is for local development)

Reply:
you can move this change to another changelist to have it locally but not comit